website/src/css/components/_Footer.scss

```diff
@@ -39,4 +43,84 @@
             padding-right: $navbar-nav-link-padding-x * 0.5;
         }
     }
+
+    &__newsletter-form-heading {
+        opacity: 0.4;
+        text-transform: uppercase;
+        font-size: 0.9rem;
+        font-weight: bolder;
+    }
+}
+.subscribe-form {
```

Comment:
Are these styles meant to be at the global level or should they be scoped to the footer?

Reply:
I'm considering having it available for blog pages, but not yet sure what the style will be or look like overall.

Reply:
If that's the case, then I'd move it into its own stylesheet.